src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,8 @@
+$start-day:
+  "mon" 0,
+  "tue" 103px,
+  "wed" 206px,
+  "thu" 309px,
+  "fri" 412px,
+  "sat" 515px,
+  "sun" 618px;
```

Comment:
better to calculate it with a formula, because you may need to change day sizes and you will need to recalculate it all. These variables were ok